github/event_types_test.go

```diff
@@ -8359,6 +8359,7 @@ func TestWorkflowRunEvent_Marshal(t *testing.T) {
 			HeadBranch: String("hb"),
 			HeadSHA:    String("hs"),
 			RunNumber:  Int(1),
+			RunAttempt: Int(1),
```

Comment:
Please revert this file... this somehow got merged in from another PR and should not be in this one.